recipes/approvaltests.cpp/8.0.0/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 2.8)
+PROJECT(test_package)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+CONAN_BASIC_SETUP()
+
+ADD_EXECUTABLE(test_package test_catch.cpp)
+#ADD_EXECUTABLE(test_package test_gtest.cpp)
```

Comment:
pls remove commented code
(or alternatively, compile executable according to the option activated)

Reply:
@SSE4  I have those in as a quick way to test against the alternate unit test frameworks. The problems is each test_framework provides its own `main`. Is there a way to build the test package to test all three? Maybe multiple directories/executables?

Reply:
take a look at boost recipe:
https://github.com/conan-io/conan-center-index/blob/master/recipes/boost/all/test_package/conanfile.py#L17
https://github.com/conan-io/conan-center-index/blob/master/recipes/boost/all/test_package/CMakeLists.txt#L34
the idea is that you pass options to the CMake from the test package conanfile.
then, in CMake, you may control which executable(s) to build (or which sources to use)